pkg/util/cert/cert.go

```diff
@@ -38,8 +38,6 @@ const (
 // +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;update
 // +kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=mutatingwebhookconfigurations,verbs=get;list;watch;update
 // +kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=validatingwebhookconfigurations,verbs=get;list;watch;update
-// +kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=validatingadmissionpolicies,verbs=get;list;watch
```

Comment:
Any reason why these roles were put in cert rotations?

Reply:
This was added to give Kueue controller permissions when errors were coming with k8s 1.31. Turns out the root cause for it is from the API server rather than cert controller ([ref](https://github.com/kubernetes-sigs/kueue/pull/3190)).